lab_1_keywords_tfidf/main.py

```diff
@@ -66,7 +96,16 @@ def get_top_n(frequencies: dict[str, Union[int, float]], top: int) -> Optional[l
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not (isinstance(frequencies, dict) and isinstance(top, int) and top > 0 and frequencies and isinstance(top, bool) == False):
+        return None
+    items = frequencies.items()
+    for key, value in items:
+        if not isinstance(key, str):
+            return None
+        if not (isinstance(value, int) or isinstance(value, float)):
+            return None
+        sorted_items = dict(sorted(items, key=lambda item: item[1], reverse=True))
+        return list(sorted_items.keys())[0:top]
```

Comment:
using this level of identation, you will return a list after validating just the first element in items